nix/mkNeovim.nix

```
@@ -97,11 +97,17 @@ with lib;
       '';
 
       installPhase = ''
-        cp -r after $out/after
-        rm -r after
         cp -r lua $out/lua
         rm -r lua
-        cp -r * $out/nvim
+        # Copy nvim/after only if it exists
+        if [ -d "after" ]; then
+            cp -r after $out/after
+            rm -r after
+        fi
+        # Copy rest of nvim/ subdirectories only if they exist
+        if [ ! -z "$(ls -A)" ]; then
+            cp -r -- * $out/nvim
```

question: why the `--`.

Shouldn't it be

```suggestion
            cp -r * $out/nvim
```

---

> question: why the `--`.
> 
> Shouldn't it be

shellcheck was complaining while i was iterating on it so i must have changed it to silence the lint.

https://www.shellcheck.net/wiki/SC2035

i don't think anyone would really name any of their config files with '-' prefix so it doesn't really matter :P

---

That's fine